cmd/containerd-shim-runhcs-v1/task_hcs.go

```diff
@@ -209,6 +211,12 @@ func newHcsTask(
 		shimOpts = v.(*runhcsopts.Options)
 	}
 
+	// For Isolated Job containers, we need special handling wherein if the command line
+	// is not specified, then we add 'cmd /C' by default.
+	if oci.IsIsolatedJobContainer(s) {
+		handleProcessArgsForIsolatedJobContainer(s, s.Process)
```

Comment:
Why is this part of it special for HPC? I dont get it, this is true for all Windows containers right? At this point its just another container